includes/reader-revenue/woocommerce/class-woocommerce-cover-fees.php

```diff
@@ -269,9 +269,12 @@ public static function get_fee_value( $subtotal ) {
 	 * @return string
 	 */
 	public static function get_fee_percentage( $subtotal ) {
-		$total = self::get_total_with_fee( $subtotal );
-		// Just one decimal place, please.
-		$flat_percentage = (float) number_format( ( ( $total - $subtotal ) * 100 ) / $subtotal, 1 );
+		$flat_percentage = 0;
+		if ( $subtotal > 0 ) {
```

Comment:
very NIT: but since we're here, why not also check if this is actually a number

Reply:
Sure thing! Done in https://github.com/Automattic/newspack-plugin/pull/3452/commits/846bdc2bb47e2eaa4101c3b5af69baddd704bfb4